docs/installation/binary-tarball.md

```diff
@@ -2,10 +2,29 @@
 
 !!! note
 
-    The following instructions install Percona Server for MySQL 5.7. 
+    The following instructions install Percona Server for MySQL 5.7.
     The instructions to install [Percona Server for MySQL 8.0 are available at this location](https://docs.percona.com/percona-server/latest/installation.html#installing-percona-server-from-a-binary-tarball).
 
-In _Percona Server for MySQL_ 5.7.31-34 and later, the multiple binary 
+## Download a {{post}} binary tarball
+
+You can download the binary tarball for Percona Server for MySQL release in the {{post}} program using your `CLIENTID` and `TOKEN`.
+
+| Type    | Name                                                                |Description         |
+|---------|---------------------------------------------------------------------|--------------------|
+| Full    | Percona-Server-<release>/private/[CLIENTID]-[TOKEN]/Percona-Server-5.7/Percona-Server-{{release}}/binary/tarball/Percona-Server-{{release}}-Linux.x86_64.glibc2.17.tar.gz   | Contains binaries, libraries, test files, and debug symbols   |
```

Comment:
The <release> variables does not work

This one `Percona-Server-<release>/private/ `looks like `Percona-Server-/private/` in preview